src/main.c

```diff
@@ -200,6 +200,9 @@ static int main_loop(libpd_cfg_t *cfg, char *firewall_cmd, char *data_file, char
     int max_retry_sleep = (1 << 9) - 1;
     int c = 2;
 
+    // ToDo: Whhy id firewall_cmd not used here?
+    (void ) firewall_cmd;
```

Comment:
It should not be called here, because it is being used here - https://github.com/Comcast/aker/pull/68/files#diff-803c5170888b8642f2a97e5e9423d399R130

Reply:
??? Why is it being passed to main loop then?

Reply:
Good question. Maybe not necessary to be passed to _main_loop_ .

Reply:
Remove this later with a different PR.

Reply:
I did not add it, just put a void cast on it since I was getting a compiler error on it (unused var)
I will remove it later on.